recipes/raspa2/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+mkdir m4
+aclocal
+autoreconf -i
+automake --add-missing
+autoconf
+./configure --prefix=${PREFIX}
+make
+make install
```

Comment:
Is there a `make check` we can run here?

Reply:
thanks for checking @ocefpaf - there is, but it currently doesn't do anything
```
$ make check
Making check in src
make[1]: Nothing to be done for `check'.
Making check in forcefield
Making check in GenericMOFs
make[2]: Nothing to be done for `check'.
Making check in GenericZeolites
make[2]: Nothing to be done for `check'.
Making check in CrystalGenerator
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-10
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-16
make[2]: Nothing to be done for `check'.
Making check in GarciaPerez2006
make[2]: Nothing to be done for `check'.
Making check in CoreShellSchroderSauer
make[2]: Nothing to be done for `check'.
Making check in CoreShellSastreGale2005
make[2]: Nothing to be done for `check'.
Making check in CoreShellCatlow
make[2]: Nothing to be done for `check'.
Making check in RigidIonKramer1991
make[2]: Nothing to be done for `check'.
Making check in Pascual2004
make[2]: Nothing to be done for `check'.
Making check in AmirjalayerSchmid
make[2]: Nothing to be done for `check'.
Making check in Nicholas
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2012MIL-100
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2012MIL-101
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-IRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-MgMOF-74
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-MgMOF-74-extended
make[2]: Nothing to be done for `check'.
Making check in Dzubak2012-ZnMOF-74
make[2]: Nothing to be done for `check'.
Making check in MgMOF-74-Yazaydin
make[2]: Nothing to be done for `check'.
Making check in CastilloVlugtCalero2009
make[2]: Nothing to be done for `check'.
Making check in TraPPE
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in molecules
Making check in TraPPE
make[2]: Nothing to be done for `check'.
Making check in TraPPE-UA
make[2]: Nothing to be done for `check'.
Making check in TraPPE-EH
make[2]: Nothing to be done for `check'.
Making check in CastilloVlugtCalero2009
make[2]: Nothing to be done for `check'.
Making check in EPM2
make[2]: Nothing to be done for `check'.
Making check in Generic
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in framework
Making check in Dubbeldam2007FlexibleIRMOF-1
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-10
make[2]: Nothing to be done for `check'.
Making check in Dubbeldam2007FlexibleIRMOF-16
make[2]: Nothing to be done for `check'.
Making check in CoreShellCatlow
make[2]: Nothing to be done for `check'.
Making check in CoreShellSchroderSauer
make[2]: Nothing to be done for `check'.
Making check in CoreShellSastreGale2005
make[2]: Nothing to be done for `check'.
Making check in Demontis
make[2]: Nothing to be done for `check'.
Making check in DemontisModified
make[2]: Nothing to be done for `check'.
Making check in Nicholas
make[2]: Nothing to be done for `check'.
Making check in NicholasModified
make[2]: Nothing to be done for `check'.
Making check in AmirjalayerSchmid
make[2]: Nothing to be done for `check'.
make[2]: Nothing to be done for `check-am'.
Making check in structures
Making check in zeolites
Making check in cif
make[3]: Nothing to be done for `check'.
Making check in block
make[3]: Nothing to be done for `check'.
Making check in ions
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in mofs
Making check in cif
make[3]: Nothing to be done for `check'.
Making check in block
make[3]: Nothing to be done for `check'.
Making check in ions
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in minerals
Making check in cif
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
Making check in ceramics
Making check in cif
make[3]: Nothing to be done for `check'.
make[3]: Nothing to be done for `check-am'.
make[2]: Nothing to be done for `check-am'.
make[1]: Nothing to be done for `check-am'.
```

I did open an issue concerning a test suite to run https://github.com/iRASPA/RASPA2/issues/12, I suggest that I will include some tests once this has been resolved.
(so far, the recipe just runs `simulate -h`)

Reply:
That is a good strategy to make all test pass!